Assets/Engine/Utilities/NoisyCollision.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NoisyCollision : MonoBehaviour
```

Comment:
This class has very similar code to the one in `Item`. Is it not possible to just add this script to items instead of duplicating the code in the `Item` class?

Reply:
The duplicated code in the item class is unorthodox but there was interest in redoing the audio system later to avoid having an overabundance of components on objects. If my code will be scrubbed in the future, it would just be easier to erase the code from the items script later than to chase down all instances of the Noisy Collision script to delete them. Was Seteron's suggestion.

Noisy Collision as a standalone script may be helpful in the case that things that aren't necessarily items may also want collision noises -- such as ragdolling entities.